raiden-network/light-client
# Bugfix/dapp service worker cache and update

Fixes #2414

**Short description**
This is just a tiny PR that will become followed by a much bigger one. I basically just fixes the no more working service worker update event and adds some actual pre-caching to it.
This does not fulfill the requirements by the management of an independent offline version with manual update process. This will still fallback to the network and also does apply updates automatically on the next run of the web-browser.

**Definition of Done**

- ~~ Acceptance criteria are met~~

**Steps to manually test the change (dApp)**

1.1 Build the dApp and serve it locally
2.1. Access the dApp in the browser and wait 1-2 seconds
2.2. Stop server and try to reload the page
2.3. Make sure it works
2.4. Rebuild the dApp and serve again
2.5. Verify that the update notification appears again (checks for updates all 60s)
